### PR TITLE
Fixed Error Raised Due to Wrongly Accessing Training Sample

### LINKED ISSUE #16114: Issue in Tutorial - "Fine-tune a pretrained model"
One of the code snippets in the tutorial are -

```
from datasets import load_dataset

dataset = load_dataset("yelp_review_full")
dataset[100]
```

However this raises an error - 

```
KeyError                                  Traceback (most recent call last)
[<ipython-input-1-0e4a64458134>](https://localhost:8080/#) in <module>()
      2 
      3 dataset = load_dataset("yelp_review_full")
----> 4 dataset[100]

[/usr/local/lib/python3.7/dist-packages/datasets/dataset_dict.py](https://localhost:8080/#) in __getitem__(self, k)
     46             suggested_split = available_suggested_splits[0] if available_suggested_splits else list(self)[0]
     47             raise KeyError(
---> 48                 f"Invalid key: {k}. Please first select a split. For example: "
     49                 f"`my_dataset_dictionary['{suggested_split}'][{k}]`. "
     50                 f"Available splits: {sorted(self)}"

KeyError: "Invalid key: 100. Please first select a split. For example: `my_dataset_dictionary['train'][100]`. Available splits: ['test', 'train']"
```

I believe this is just a missing extra key while trying to access the value in the dictionary. 

So the last line should be something like - 

`dataset['train'][100]`

Which does indeed give a correct output - 

```
{'label': 0,
 'text': 'My expectations for McDonalds are t rarely high. But for one to still fail so spectacularly...that takes something special!\\nThe cashier took my friends\'s order, then promptly ignored me. I had to force myself in front of a cashier who opened his register to wait on the person BEHIND me. I waited over five minutes for a gigantic order that included precisely one kid\'s meal. After watching two people who ordered after me be handed their food, I asked where mine was. The manager started yelling at the cashiers for \\"serving off their orders\\" when they didn\'t have their food. But neither cashier was anywhere near those controls, and the manager was the one serving food to customers and clearing the boards.\\nThe manager was rude when giving me my order. She didn\'t make sure that I had everything ON MY RECEIPT, and never even had the decency to apologize that I felt I was getting poor service.\\nI\'ve eaten at various McDonalds restaurants for over 30 years. I\'ve worked at more than one location. I expect bad days, bad moods, and the occasional mistake. But I have yet to have a decent experience at this store. It will remain a place I avoid unless someone in my party needs to avoid illness from low blood sugar. Perhaps I should go back to the racially biased service of Steak n Shake instead!'}
```

### PR DESCRIPTION
**Pull request by aflah02**
Fixed Error Raised Due to Wrongly Accessing Training Sample

# What does this PR do?

Fixes [16114](https://github.com/huggingface/transformers/issues/16114)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by patil-suraj — APPROVED** [review]
Thanks for fixing this! Could you run `make style` and push again, that will fix the failing tests.

**Comment by aflah02** [comment]
Hey
@patil-suraj I did run make style but I see lots of files have now changed not sure if this is the intended effect but all I did was git clone an editable pip install and make style

**Comment by LysandreJik** [comment]
Hey @aflah02, it looks like an issue with windows and line-endings. Could you revert your last commit, and we can take care of fixing the quality if you give us write access on your repository!

**Comment by aflah02** [comment]
Sure @LysandreJik I'll revert the last commit, who should I give write access to?

**Comment by patil-suraj** [comment]
> Sure @LysandreJik I'll revert the last commit, who should I give write access to?

Hey @aflah02 ! There's a button at the very bottom on the right hand panel of this page, called `Allow edits and access to secrets by maintainers`, if you tick that box any of us Transformers maintainers can access this PR.

**Comment by aflah02** [comment]
Hey @patil-suraj I don't see one by that name but there's one by the name "Allow edits by maintainers" which is already checked, is that it?

File: docs/source/training.mdx [hunk 1]
```diff
@@ -34,7 +34,7 @@ Begin by loading the [Yelp Reviews](https://huggingface.co/datasets/yelp_review_
 >>> from datasets import load_dataset
 
 >>> dataset = load_dataset("yelp_review_full")
->>> dataset[100]
+>>> dataset['train'][100]
```
Comment (patil-suraj) [inline docs/source/training.mdx:37]: This should fix the failing test
```suggestion
>>> dataset["train"][100]
```

**Comment by aflah02** [comment]
Thanks!

**Pull request merged**